CMakeLists.txt

```diff
@@ -418,11 +419,14 @@ if (WIN32)
 endif(WIN32)
 # Definition directives
 
-# The following defines were extracted from the official amd64 debug / release builds.
 if (WIN32)
   add_definitions(-DFEATURE_EVENT_TRACE=1)
 endif (WIN32)
 
+if (CLR_CMAKE_PLATFORM_LINUX)
```

Comment:
I'd prefer using "OR" in the condition instead of haveing two ifs for the same definition.